### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by slesaad**
# Changes
- Use svg marker in lieu of the golden png marker, update color and add fill to the circle
- Update map style that uses state/city labels and make sure the labels are in the correct layer order
- Add projection to map so it's not 3d anymore
- Update polygon color to match ghg blue
- Update vulcan data to recent and use different rescale values (and thus legend update)
- Update default map zoom so it fits the whole US in embedded view

### CODE REVIEW

File: urban-dashboard/src/components/mapboxViewer/index.jsx [hunk 1]
```diff
@@ -35,11 +35,14 @@ export class MapBoxViewer extends Component {
 
         const map = new mapboxgl.Map({
             container: 'mapbox-container',
-            renderingMode: '2D',
+            projection: "mercator",
             style: mapboxStyleUrl,
             center: [-99.676392, 39.106667], // Center of the USA
-            zoom: 4.8, // Adjust zoom level to fit the USA
-            zoomControl: true
+            zoom: 4, // Adjust zoom level to fit the USA
```
Comment (slesaad) [inline urban-dashboard/src/components/mapboxViewer/index.jsx:41]: this works better in embedded view

File: urban-dashboard/public/marker.svg
Comment (sanzog03) [inline urban-dashboard/public/marker.svg:1]: We could use the SVG marker that is used in the NIST interface. No more marker asset addition, as you can just change the color for the needed marker.

File: urban-dashboard/src/components/mapboxViewer/helper/index.jsx
Comment (sanzog03) [inline urban-dashboard/src/components/mapboxViewer/helper/index.jsx:1]: Add an option to take these urls via env rather than hard coding. Can leave these as default value though! 
This would be suitable for a multi-environment setup (dev, stag, prod). Also if we do so, we would be able to change it without another commit.

File: urban-dashboard/src/components/mapboxViewer/index.css
Comment (sanzog03) [inline urban-dashboard/src/components/mapboxViewer/index.css:1]: Using the SVG will help in minimizing the bundle size as well.

File: urban-dashboard/src/components/mapboxViewer/index.jsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ export class MapBoxViewer extends Component {
 
     plotMap() {
         mapboxgl.accessToken = accessToken;
-        let mapboxStyleUrl = 'mapbox://styles/mapbox/satellite-v9';
+        let mapboxStyleUrl = 'mapbox://styles/covid-nasa/cldu1cb8f00ds01p6gi583w1m';
```
Comment (sanzog03) [inline urban-dashboard/src/components/mapboxViewer/index.jsx:26]: Take this from env

File: urban-dashboard/src/components/mapboxViewer/index.jsx (hunk 1 above)
Comment (sanzog03) [reply urban-dashboard/src/components/mapboxViewer/index.jsx:40]: If possible could we add an ability to pass in zoom level as a query params? It will be very handy to fine-tune zoom levels according to the viewer's size. (ref. NIST interface). Not a necessary one but a nice to have one.

**Review by anisbhsl — APPROVED** [review]

File: urban-dashboard/src/components/mapboxViewer/index.jsx (hunk 2 above)
Comment (anisbhsl) [reply urban-dashboard/src/components/mapboxViewer/index.jsx:26]: Is this the style for 3d or 2d?

File: urban-dashboard/public/marker.svg
Comment (slesaad) [reply urban-dashboard/public/marker.svg:1]: i copied over the same svg but filled the inside with white because otherwise no color looks good with that raster layer 🤷‍♀️

File: urban-dashboard/src/components/mapboxViewer/index.jsx (hunk 2 above)
Comment (slesaad) [reply urban-dashboard/src/components/mapboxViewer/index.jsx:26]: this is for the city labels and the one that the dashboard uses
it supports 3d, but by setting projection to mercator, we can make it 2d

File: urban-dashboard/public/marker.svg
Comment (sanzog03) [reply urban-dashboard/public/marker.svg:1]: we could use the same yellowish/gold color in the SVG (basically the one used right now but with no gradient)?
